src/Compilers/Core/Portable/Symbols/ISymbolExtensions_PerformIVTCheck.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Collections;
+
+namespace Microsoft.CodeAnalysis
```

Comment:
Nit: Should the filename use a dot as a separator instead of underscore, to be consistent with rest of codebase?